static/app/views/replays/detail/network/index.tsx

```diff
@@ -181,12 +196,14 @@ const StatusPlaceHolder = styled(Placeholder)`
   max-width: 40px;
 `;
 
-const Text = styled('p')`
+const Text = styled('p')<{dimmed?: boolean; italic?: boolean}>`
   padding: 0;
   margin: 0;
   text-overflow: ellipsis;
   white-space: nowrap;
   overflow: hidden;
+  ${p => p.italic && 'font-style: italic;'}
+  ${p => p.dimmed && `color: ${p.theme.subText};`}
 `;
```

Comment:
Instead of introducing props and creating dynamic styles for this styled component, it would be better to extend and create a new component

```suggestion
const EmptyText = styled(Text)`
  font-style: italic;
  color: ${p => p.theme.subText};
`;
```